fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ConcreteAvailability.java

```diff
@@ -105,4 +105,8 @@ public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constra
     protected Map<String, List<Interval>> getLatestAvailableIntervalsByTable() {
         return metadataService.getAvailableIntervalsByTable(name);
     }
+
+    public Set<String> getColumnNames() {
```

Comment:
I think this method is a bit dull to be here, remove it makes less code to maintain without losing basically anything.